cli.js

```diff
@@ -297,7 +297,7 @@ if (inputDir || outputDir) {
   processDirectory(inputDir, outputDir, fileExt);
 }
 // Minifying one or more files specified on the CMD line
-else if (typeof content === 'string') {
+else if (content || content.length > 0) {
```

Comment:
Wouldn't `else if (content) {` suffice here? `''` is falsy after all.

Reply:
Yup! I'll get that changed.